backend/app-kubernetes-service/api/src/main/kotlin/api/KubernetesLicense.kt

```diff
@@ -21,7 +21,13 @@ data class KubernetesLicense(
     val address: String,
     val port: Int,
     val tags: List<String>,
-    val license: String?
+    val license: String?,
+    val category: ProductCategoryId,
+    val pricePerUnit: Long,
+    val description: String,
+    val availability: ProductAvailability,
+    val priority: Int,
+    val paymentModel: PaymentModel = PaymentModel.PER_ACTIVATION,
```

Comment:
Should some of these use default values? Should paymentModel default be removed?

Reply:
I would try to mirror what the `Product`s already do in terms of default values.